packages/node-bridge/test/bridge.test.js

```diff
@@ -29,21 +29,34 @@ test('`NowProxyEvent` normalizing', async () => {
       })
     )
   );
-  const bridge = new Bridge(server);
+
+  let features;
+
+  class CustomBridge extends Bridge {
+    handleEvent(normalizedEvent) {
+      features = normalizedEvent.features;
+      return super.handleEvent(normalizedEvent);
+    }
+  }
+
+  const bridge = new CustomBridge(server);
   bridge.listen();
+
   const context = { callbackWaitsForEmptyEventLoop: true };
   const result = await bridge.launcher(
     {
       Action: 'Invoke',
       body: JSON.stringify({
         method: 'POST',
         headers: { foo: 'baz' },
+        features: { enabled: true },
         path: '/nowproxy',
         body: 'body=1',
       }),
     },
     context
   );
+  assert.strictEqual(features.enabled, true);
```

Comment:
```suggestion
  assert.deepStrictEqual(features, { enabled: true });
```

Lets check the whole object